raisinbread/README.md

```diff
@@ -50,12 +50,21 @@ cat raisinbread/instruments/instrument_sql/9999-99-99-drop_instrument_tables.sql
 
 ***Important:** Ensure that the above commands were completed properly and that all the tables were dropped before continuing the installation process.*
 
+If you could not drop all the tables with the commands above, you have the option to drop the tables by  manually sourcing these .sql files directly 
+from the mysql command line as follows:
+```bash
+mysql> source /var/www/loris/raisinbread/instruments/instrument_sql/9999-99-99-drop_instrument_tables.sql
```

Comment:
I suggest to remove the path /var/www/loris/ (next line also). It's setup dependent and as the next commands below are omitting it, we suppose the user cd there first.

Reply:
Thanks for the advice.